pipelines/build/common/weekly_release_pipeline.groovy

```diff
@@ -43,6 +43,7 @@ stage("Submit Release Pipelines") {
                       parameters: [
                           string(name: 'releaseType',        value: 'Release'),
                           string(name: 'scmReference',       value: scmRef),
+                          string(name: 'aqa-testsReference',   value: scmRef),
```

Comment:
aqaTestsReference (same comment for other occurrences as well)

Reply:
Removed. The change is not necessary.